app/routes/search/agendaitems.js

```diff
@@ -69,6 +68,13 @@ export default class AgendaitemSearchRoute extends Route {
         entry.shortTitle = entry.shortTitle.replaceAll(highlightShortTitleEmsRemoved, agendaitem.highlight.shortTitle[i]);
       }
     } 
+    if ((agendaitem.highlight && agendaitem.highlight.title) && agendaitem.highlight.title[0]?.split(" ").length < agendaitem.attributes.title?.split(" ").length ) {
+      entry.title = agendaitem.attributes.title;
+      for (let i = 0; i < agendaitem.highlight.title.length; i++){
```

Comment:
I would consider using a `for ... of` loop. `for (const titleHighlight of agendaitem.highlight.title)`